bricksrc/sensor.py

```diff
@@ -1666,6 +1666,63 @@
                             },
                         },
                     },
+                    "Radiant_Panel_Temperature_Sensor": {
+                        "tags": [
+                            TAG.Point,
+                            TAG.Sensor,
+                            TAG.Temperature,
+                            TAG.Radiant,
+                            TAG.Panel,
+                        ],
+                        "substances": [
+                            [BRICK.measures, BRICK.Temperature],
+                            [BRICK.measures, BRICK.Radiant_Panel],
+                        ],
+                        "subclasses": {
+                            "Inside_Face_Surface_Temperature_Sensor": {
+                                "tags": [
+                                    TAG.Point,
+                                    TAG.Inside,
+                                    TAG.Face,
+                                    TAG.Surface,
+                                    TAG.Temperature,
+                                    TAG.Sensor,
+                                ],
+                            },
+                            "Outside_Face_Surface_Temperature_Sensor": {
+                                "tags": [
+                                    TAG.Point,
+                                    TAG.Outside,
+                                    TAG.Face,
+                                    TAG.Surface,
+                                    TAG.Temperature,
+                                    TAG.Sensor,
+                                ],
+                            },
+                            "Embedded_Temperature_Sensor": {
+                                "tags": [
+                                    TAG.Point,
+                                    TAG.Embedded,
+                                    TAG.Temperature,
+                                    TAG.Sensor,
+                                ],
+                                "substances": [
+                                    [BRICK.measures, BRICK.Temperature],
+                                    [BRICK.measures, BRICK.Internal_Material],
+                                ],
+                                "subclasses": {
+                                    "Core_Temperature_Sensor": {
+                                        "tags": [
+                                            TAG.Point,
+                                            TAG.Core,
+                                            TAG.Temperature,
+                                            TAG.Sensor,
+                                        ],
+                                    },
```

Comment:
A few comments:
- Given that "embedded" is an overloaded term (it may refer to a temperature sensor on an "embedded" platform such as a sensor mote), what might be a clearer word we can use for Embedded (and Core) temperature sensors? I understand these may be "terms of art", so maybe something like `Embedded_Panel_Temperature_Sensor` and `Core_Panel_Temperature_Sensor` (or `Panel_Embedded...`, etc)?

Reply:
Good point! I need to think about this. I think the term needs to capture that it is a temperature measurement inside the material but not necessarily belonging to a radiant system.